homeassistant/components/notify/aws_sqs.py

```diff
@@ -23,15 +23,15 @@
 
 PLATFORM_SCHEMA = vol.Schema({
     vol.Required(CONF_PLATFORM): "aws_sqs",
-    vol.Optional(CONF_NAME): vol.Coerce(str),
+    vol.Required(CONF_NAME): vol.Coerce(str),
```

Comment:
No need to require a name, we just fallback to the platform name (in this case `aws_sqs`)